src/runtime/contrib/tensorrt/tensorrt_builder.cc

```diff
@@ -209,7 +209,7 @@ runtime::TrtEngineAndContext TensorRTBuilder::BuildEngine(
            num_input_bindings + network_output_names_.size());
   nvinfer1::IExecutionContext* context = engine->createExecutionContext();
   return {engine, context, network_input_names_, network_input_is_baked_,
-          network_output_names_};
+          network_output_names_, std::vector<void*> (engine->getNbBindings(), nullptr)};
```

Comment:
We could actually allocate the buffers here, since we have access to the input arg DLTensors here (class member `execution_args_`). I think this would be slightly cleaner, and would remove any chance of double allocation.

Reply:
Done.